packages/client/src/views/Performance/RegistrationRates.tsx

```diff
@@ -147,7 +168,12 @@ function RegistrationRatesComponent(props: IRegistrationRateProps) {
             return (
               <Within45DaysTable
```

Comment:
@Sadman-Ilham, this where you should include your component only if `base.baseType === REG_RATE_BASE.TIME`